molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/usermanager/UserManagerServiceImpl.java

```diff
@@ -41,6 +44,12 @@ public void setActivationUser(String userId, Boolean active) {
     this.dataService.update(USER, user);
   }
 
+  @Override
+  @PreAuthorize("hasAnyRole('ROLE_SU','ROLE_MANAGER')")
```

Comment:
This check will only go off on a server where a non-su non-manager user is given plugin permissions on the user manager. In that rare case it'll cause a nasty 500 error breaking the complete screen.
I'd say simply remove the check.

Reply:
Nevermind, the rest of the methods have the same annotation.